site-modules/adhoc/tasks/nix_example.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+# script content goes here
```

Comment:
In Bolt, scripts and tasks are different. Scripts go in scripts/. This is a task, so comment should use that language.
```suggestion
# task content goes here
```